plugins/c8y_remote_access_plugin/src/main.rs

```diff
@@ -184,6 +189,25 @@ fn build_proxy_url(cumulocity_host: &str, key: &str) -> miette::Result<Url> {
         .context("Creating websocket URL")
 }
 
+fn get_executable_path(config_dir: &Utf8Path) -> miette::Result<PathBuf> {
+    let operation_path = supported_operation_path(config_dir);
+
+    let content = std::fs::read_to_string(&operation_path)
+        .into_diagnostic()
+        .context(format!(
```

Comment:
```suggestion
        .with_context(|| format!(
```

This means that we construct the string lazily when an error occurs rather than every time we call this function.